src/mlpack/bindings/python/is_serializable.hpp

```diff
@@ -43,8 +43,8 @@ void IsSerializable(util::ParamData& data,
       IsSerializable<typename std::remove_pointer<T>::type>(data);
 }
 
-} // python
-} // bindings
-} // mlpack
+} // namespace python
+} // namespace bindings
+} // namespace mlpack
 
```

Comment:
Are we okay with removing the namespace word?

Reply:
Ignore my above comment, I was looking at the old code, it seems that the linter has added it, I was surprised that we did not have the namespace word, the code was probably too old. :+1: